i18n/locales/de/common.json

```diff
@@ -1,238 +1,532 @@
+  "Learn about blockchain with our comprehensive knowledge base.": "Erfahre mehr über Blockchain mit unserer umfassenden Wissensdatenbank.",
+  "Learn more": "Erfahre mehr",
+  "Ledger Nano S Connected.": "Ledger Nano S verbunden",
+  "Ledger Nano S Disconnected.": "Ledger Nano S getrennt",
+  "Like almost all blockchains Lisk has transparency at it’s core. This means that anyone can view everything that happens on the Lisk network, including the holdings of each account.": "Wie bei fast allen Blockchains ist Transparenz bei Lisk grundlegend. Das bedeutet, dass jeder alles was im Lisk-Netzwerk passiert einsehen kann, einschließlich der Bestände der einzelnen Konten.",
+  "Lisk Academy": "Lisk Academy",
   "Lisk Chat": "Lisk Chat",
   "Lisk Explorer": "Lisk Explorer",
-  "Lisk Forum": "Lisk Forum",
+  "Lisk Help Center": "Lisk Help Center",
   "Lisk Hub": "Lisk Hub",
-  "Lisk Website": "Lisk Webseite",
-  "Login": "Einloggen",
-  "Losing access to this passphrase will mean no funds can be sent from this account.": "Das Verlieren der Passphrase führt dazu, dass kein Geld mehr von diesem Konto versendet werden kann.",
-  "Mainnet": "Mainnet",
-  "Maximum of {{n}} votes exceeded.": "Das Maximum an {{n}} Stimmen wurde überschritten.",
-  "Maximum of {{n}} votes in one transaction exceeded.": "Das Maximum an {{n}} Stimmen in einer Transaktion wurde überschritten.",
+  "Lisk Hub currently supports Ledger Nano S, Ledger Nano X and Trezor Model T wallets": "Lisk Hub unterstützt derzeit Ledger Nano S, Ledger Nano X und Trezor Model T Wallets.",
+  "Lisk Hub {{version}}": "Lisk Hub {{Version}}",
+  "Lisk Hub {{version}} is here!": "Lisk Hub {{Version}} is da!",
+  "Lisk Terms of Use": "Lisk Nutzungsbedingungen",
+  "Lisk Website": "Lisk Website",
+  "Lisk accounts on {{WalletModel}}": "Lisk Konten auf {{WalletModel}}",
+  "Lisk's blockchain is based on the Delegated Proof of Stake (DPoS) consensus algorithm, in which 101 delegates are voted in by token holders to secure the network.": "Die Blockchain von Lisk basiert auf dem Konsensalgorithmus des Delegated Proof of Stake (DPoS), bei dem 101 Delegierte von Token-Inhabern zur Sicherung des Netzwerks gewählt werden.",
+  "Lisk.Chat": null,
+  "Load more": "Mehr laden",
+  "Loading": "Laden",
+  "Loading name...": "Ladenname",
+  "Locale": "Locale",
+  "Log out automatically after 10 minutes.": "Nach 10 Minuten automatisch abmelden.",
+  "Login Type not recognized.": "Anmeldungsart nicht erkannt.",
+  "Look at your Ledger for confirmation": "Sehen das Ledger zur Bestätigung an.",
+  "Look at your Trezor %s for completing the action": "Schau dir deine Trezor % an, um die Aktion abzuschließen.",
+  "Looking for a device...": "Auf der Suche nach einem Gerät.....",
+  "Low": "Niedrig",
+  "Mainnet": "Hauptnetz",
+  "Manage your LSK with ease": "Verwalten dein LSK mit Leichtigkeit.",
+  "Manage your most used accounts": "Verwalten deiner meistgenutzten Konten",
+  "Max": "Maximum (or Höchstwert )",
```

Comment:
```suggestion
  "Max": "Maximum",
```
I think ` (or Höchstwert )` was a note from the translator
And it doesn't fit in the UI anyway:
![Screenshot 2019-09-02 at 15 09 29](https://user-images.githubusercontent.com/1254342/64116848-b90c7a00-cd93-11e9-9b72-c0272666a4a5.png)